src/ast-utils/docs.ts

```diff
@@ -135,11 +138,14 @@ function copyProperties(from, to, behaviorsByName) {
   if (!from.behaviors) {
```

Comment:
Line 133 captures a bug fixed in: https://github.com/Polymer/hydrolysis/pull/211